src/store/store.ts

```
@@ -112,12 +112,25 @@
+  private static _instance: Store;
+  public static getInstance(): Store {
+    if (!Store._instance) {
+      Store._instance = new Store();
+    }
+    return Store._instance;
+  }
+
   constructor() {
+    // Use as a singleton
+    if (Store._instance) {
+      return Store._instance;
```

Biome

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)